Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">SHAZAM!!!</h1>
+		<hr>
+		<h3 class="text-color">There's something you should know about me, Joe Rogan...
+			<i class="fa fa-hand-spock-o"></i>
+		</h3>
+
+		<p class="text-color">My name is Tyrone Biggums. I speak at elementary schools regarding the perils of using drugs. 
+ 		In my free time, I enjoy:
+ 			<ul class="text-color">
+ 				<li>Stealing radios</li>
+ 				<li>Dancing</li>
+ 				<li>Peanut butter and crack sandwiches</li>
+ 			</ul> 
+			
+		</p>
+
+		<div class="center">
+			<p>
+				<a href="https://www.linkedin.com/in/tyrone-biggums-8ba85553">
+					<img class= "animate" src="images/Tyrone Biggums BIO.png">
+				</a> 
+				
+			</p>
+			
+			<p>
+				<a href="https://twitter.com/tyrone_biggums">
+					<img class= "animate" src="images/TyroneBiggumsRedbull.jpg">
+				</a> 
+				
+			</p>
+			
+			<p>
+				<a href="https://www.facebook.com/Tyrone-Biggums-118335254862251/?fref=ts">
+					<img class= "animate" src="images/tyrone_dancing.gif">
+				</a> 
+				
+			</p>
+			<i class="fa fa-facebook"></i>
+			<i class="fa fa-twitter"></i>
+			<i class="fa fa-linkedin"></i>
+			
+		</div>
+			
```

Comment:
:hammer: Just to recap, make sure you're getting rid of unnecessary empty space. Clean code will make you stand out from the rest!